cgo_flags.go

```diff
@@ -6,9 +6,10 @@ package rocksdb
 // #cgo CPPFLAGS: -Iinternal -Iinternal/include -Iinternal/db -Iinternal/util
 // #cgo CPPFLAGS: -Iinternal/utilities/merge_operators/string_append
 // #cgo CPPFLAGS: -I../c-snappy/internal -I../c-lz4/internal/lib
-// #cgo CPPFLAGS: -DROCKSDB_PLATFORM_POSIX -DNDEBUG -DSNAPPY -DLZ4
-// #cgo darwin CPPFLAGS: -DOS_MACOSX
-// #cgo linux CPPFLAGS: -DOS_LINUX
+// #cgo CPPFLAGS: -DNDEBUG -DSNAPPY -DLZ4
+// #cgo darwin CPPFLAGS: -DOS_MACOSX -DROCKSDB_PLATFORM_POSIX
+// #cgo linux CPPFLAGS: -DOS_LINUX -DROCKSDB_PLATFORM_POSIX
+// #cgo windows CPPFLAGS: -DOS_WIN
```

Comment:
Nah, I was trying to get cross-compilation working so we could post binaries and wanted to see how far I could get with windows.